packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -22,6 +22,7 @@ import ConfirmationModal from '@salesforce/retail-react-app/app/components/confi
 import {useItemVariant} from '@salesforce/retail-react-app/app/components/item-variant'
 import {noop} from '@salesforce/retail-react-app/app/utils/utils'
 import {API_ERROR_MESSAGE} from '@salesforce/retail-react-app/app/constants'
+import {values} from 'lodash'
```

Comment:
I believe this import is unused

Reply:
I don't think I added that.... But it looks like you are correct

Reply:
Done